src/index.ts

```diff
@@ -2,6 +2,7 @@ import { createContext, runInContext, Result as SourceResult } from 'js-slang'
 import { SourceError } from 'js-slang/dist/types'
 
 const TIMEOUT_DURATION = parseInt(process.env.TIMEOUT!, 10) // in milliseconds
+require('./graphics/rune_library')
```

Comment:
Prefer ES6 `import`. Also, would prefer a function `importExternals` that uses a switch to decide what libs to import. Probably place this function call in runAll before the call to evaluateGlobals. Would prevent name conflicts when more external libs are added.

Reply:
For `.js` files, I used `require()` as well, and kept a single object/function as the default export. If `import` works here, that's even better